src/reynir/Greynir.grammar

```diff
@@ -152,8 +152,10 @@ MgrInnihald →
     Forskeyti? Yfirsetning StLiður* Lokatákn?
     | Staðhæfing
 
+# "Allt er klárt," segir Jón og bætir við: "Komum þessu í gang."
 Staðhæfing →
     Sagt Segjandi Lokatákn?
+    | Sagt Segjandi Segir SegirÁfram Sagt Lokatákn?
```

Comment:
Better style (and probably better performance) to have this as one production with an optional part:

```
Staðhæfing →
    Sagt Segjandi ÁframSagt? Lokatákn?

ÁframSagt  →
    Segir SegirÁfram Sagt

```

Reply:
Much sleeker! Updated.